src/main/java/appeng/core/api/ApiPart.java

```diff
@@ -332,4 +361,53 @@ public String map( final String typeName )
 			return o;
 		}
 	}
+
+
+	private class CacheKey
+	{
+		private final Class<? extends AEBaseTile> baseClass;
+
+		private final List<String> interfaces;
+
+		public CacheKey( Class<? extends AEBaseTile> baseClass, List<String> interfaces )
+		{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Should appeng.core.api.ApiPart$CacheKey be a _static_ inner class? [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=findbugs%3ASIC_INNER_SHOULD_BE_STATIC)